examples/experimental/ProductionMonitoring/stream_table_api.ipynb

```diff
@@ -0,0 +1,170 @@
+   "source": [
+    "# Weave StreamTable Usage\n",
+    "\n",
+    "This notebook demonstrates basic Weave StreamTable usage with interactive examples.\n",
+    "\n",
+    "## Step 0: Setup\n",
+    "\n",
+    "All the StreamTables created in this notebook will be saved to the WB_PROJECT under the WB_ENTITY account on the public W&B cloud. \n",
+    "\n",
+    "**Please login to W&B and set your WB_ENTITY** before running this demo."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "51c30e7a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import weave\n",
+    "from weave.monitoring import StreamTable"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5684f0b8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "WB_ENTITY = \n",
+    "WB_PROJECT = \"mesa\"\n",
+    "STREAM_TABLE_NAME = \"my_stream_table\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6f676d96",
+   "metadata": {},
+   "source": [
+    "## Step 1: Define a StreamTable\n",
+    "\n",
+    "StreamTable has a single required argument: the name of the StreamTable object.\n",
+    "\n",
+    "```python\n",
+    "st = StreamTable(\"stacey/mesa/my_stream_table\")\n",
+    "```\n",
+    "\n",
+    "This takes the form `my_wb_entity/my_wb_project_name/my_stream_table_name` where you can modify the component names to the relevant strings (e.g. your W&B username or shared W&B team name, a new or existing W&B project name).\n",
+    "\n",
+    "**Note**: if you are logged in on the current machine and have a ~/.netrc file saved with your W&B credentials, this will be the default entity used if you do not pass in an entity string component to the StreamTable constructor."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "25bfa9dc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "st = StreamTable(f\"{WB_ENTITY}/{WB_PROJECT}/{STREAM_TABLE_NAME}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "073d25c9",
+   "metadata": {},
+   "source": [
+    "## Step 2: Log some data\n",
+    "\n",
+    "To add rows to the StreamTable, call `.log()` on the StreamTable object. \n",
+    "`.log()` accepts a single dictionary or a list of dictionaries, where each dictionary entry corresponds to one row of the table. In each dictionary, the keys are column names and the values are the corresponding cell values.\n",
+    "\n",
+    "```python\n",
+    "st.log({\"one_column_name\" : \"value_a\", \"another_column_name\" : 7})\n",
+    "st.log([{\"one_column_name\" : \"value_b\", \"another_column_name\" : 19},\n",
+    "        {\"one_column_name\" : \"value_c\", \"another_column_name\" : 28},\n",
+    "        {\"one_column_name\" : \"value_d\", \"another_column_name\" : 36}]\n",
+    "```\n",
+    "\n",
+    "The first call to `.log()` will return a Weave Panel URL, where you can view, edit, and save the resulting StreamTable as a Weave Board, of the form:\n",
```

Comment:
@tssweeney do we plan on linking to the table view or a board view with the table? I find the table view harder to use atm

Reply:
"plan", no. But i think many of us feel that the distinction between boards and object view is confusing. Likely we will change this to be a board with hidden varbar? Something like that. In any case, We should not hard-code links in here since the link format and target may change and then we would have to remember to update this notebook.

Reply:
glad I'm not the only one confused—I think having a demo link would be very useful, I'm ok changing it, otherwise our users don't really know what a Board looks like at all or where to find it :)